server/auth.go

```diff
@@ -527,6 +529,26 @@ func (s *Server) processClientOrLeafAuthentication(c *client) bool {
 	return false
 }
 
+func getTLSAuthDCs(rdns *pkix.RDNSequence) string {
+	dcOID := asn1.ObjectIdentifier{0, 9, 2342, 19200300, 100, 1, 25}
```

Comment:
const DcOID = ... 
and outside the funciont?

Reply:
think these cannot be made into `const`